### DISCUSSION TITLE
Chia Mnemonic Invalid

### DISCUSSION THREAD

**Discussion by tksalarzai** [Chia-Network/chia-blockchain#7394] [Support]
I had 8 chia farmed in very early of its launch , i had transferred 1 xch to an exchange , after that i created a new cold wallet and transferred my remaining 7 Xch to that cold wallet , my cold wallet mnemonic phrase is with me in hard copy, Yesterday i logged into my cold wallet and transferred 1 xch more out of 7 to exchange, and then deleted that key from my GUI as security measure, Now sir today when I'm entering my Xch Cold wallet Mnemonic it's saying invalid Mnemonic again and again i tried, Even by very strictly checking each word, Please sir what is the problem please help me as my wallet have 6 xch.

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
